types/probationTeams.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface ProbationTeamsClient {
```

Comment:
this already exists in licences.d.ts

Reply:
oh no - I see you are just extracting it out.

Nothing wrong with that roContactDetailsService references the type from the probationteamclient directly - not sure if there is anything wrong with that either....